scalacheck/shared/src/main/scala/matryoshka/scalacheck/arbitrary.scala

```diff
@@ -34,7 +34,7 @@ sealed trait ArbitraryInstances0 {
     F(A)
 }
 
-trait ArbitraryInstances extends ArbitraryInstances0 {
+sealed trait ArbitraryInstances extends ArbitraryInstances0 {
```

Comment:
Yeah … I actually meant to change this. The problem is that `ArbitraryInstances0` was already `sealed` and `-Xlint` complained about an unsealed extension of a sealed trait. But there is perhaps a better approach … like maybe making `ArbitraryInstances0` `private` instead of `sealed`?